### PR TITLE
SETI-4849: Support building package on el8

### PR DESCRIPTION
**Pull request by nhatking16591**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: specs/serverspec-core.spec [hunk 1]
```diff
@@ -16,28 +19,47 @@ Source0:          %{name}.tar.gz
...
+%else
+cp -a /usr/share/gems/* $RPM_BUILD_ROOT%{install_dir}/bundle/ruby/
+sed -i s/'#!ruby -w'/'#!\/usr\/bin\/env ruby'/g $RPM_BUILD_ROOT%{install_dir}/bundle/ruby/2.5.0/gems/diff-lcs-1.3/bin/*
+%endif
 install -d $RPM_BUILD_ROOT/usr/bin
 mv ./bin/serverspec $RPM_BUILD_ROOT/usr/bin/serverspec
+%if 0%{?el7}
 sed -i 's,@@SCL_PREFIX@@,%{_scl_prefix},g' $RPM_BUILD_ROOT/usr/bin/serverspec
+%else
+sed -i 's,@@SCL_PREFIX@@,,g' $RPM_BUILD_ROOT/usr/bin/serverspec
```
Comment (ignatenkobrain) [inline specs/serverspec-core.spec:61]:
```suggestion
sed -i '/@@SCL_PREFIX@@/d' $RPM_BUILD_ROOT/usr/bin/serverspec
```

I believe it should be like this, but I did not test it :)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/serverspec-core-merge-compliance-check/8/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/compliance/job/serverspec-core-fossa-scanning-zuul-el7/1/ : FAILURE in 2m 04s (non-voting)

**Pull request merged**